### ISSUE TITLE
Minimize Gaming

### ISSUE THREAD

**Issue by lenasong** [huntallcott/PhoneAddiction#11]
I'm starting this thread for us to jot down ideas related to gaming that we have discussed previously. This includes users taking screenshot at an earlier time and more sophisticated editing of screenshots.

**Comment by lenasong** [comment]
I've noticed that the files we download from Qualtrics do not show the original time at which the images are created, and instead shows the time we download these files. 

This is similarly true for other options I've looked into, including received text messages or files sent via Dropbox [file request](https://www.dropbox.com/help/files-folders/create-file-request). I looked into the latter option because Dropbox shows the original time created/time last modified for files uploaded using a shared folder; this unfortunately is not true for file request which is more appropriate for our purposes. I'm not sure shared folder is a good alternative either, since we will have to ask all users to register for dropbox and create a folder for each participant. 

@KellyBuckley @gentzkow @huntallcott @rajbhargava thoughts and suggestions?


Given this, here is a start of list of ideas to minimize gaming:

- Check screenshots across weeks are somewhat different - to prevent someone from uploading the same low usage screenshot every week
- Confirm the time of survey submission and time of screenshot displayed are within reasonable range (to prevent user from taking a screenshot Sunday morning and using that to upload to our system on Sunday night)
- Related to the point above, users of course could change the time on their phone. However they need to guess broadly when we will be sending out the screenshot reminders. Should we randomize when we send the weekly screenshot reminders? 
- Alternatively, we could have more frequent screenshots (so there are overlaps across different days of screenshot upload). Since we will collect screenshots between the baseline and midline surveys, during which participants are less likely to game, we just need to continue to check that the screenshots are sequential over time.
- Another (cheaper) idea is to have "surprise" mid-week screenshot requests that could confirm the authenticity of screenshots taken within the 7 day range. 

It was not clear to me before testing that qualtrics does not store the original timestamps. Perhaps our participants will assume that we can easily check that and my worries above are not relevant.

**Comment by huntallcott** [comment]
@lenasong thanks for thinking about this. These are all good ideas, and honestly I don't know which is best.

**Comment by lenasong** [comment]
@huntallcott @gentzkow thanks for the feedback.

Some quick comments:

- That was my prior as well that the photos should include creation date as part of EXIF metadata. I've tested with two screenshots and then downloaded them from Qualtrics - they are saved under Dropbox\PhoneAddiction\Pilot\Test Screenshots. From my computer (Windows), the origin information is blank. Perhaps someone else has a better way of accessing this information?

- We can definitely use the time recording functions in Qualtrics. I'll also add timers to each question so we can look at the amount of time spent on the screenshot upload questions (doing survey then switch into Screen Time and taking screenshots will presumably take a bit longer than just uploading prepared screenshots directly).

- I've added the following to the survey in the spirit of tough talk: Any manipulation of screenshots will result in no payment and immediately disqualify you from any participation in future studies.

**Comment by gentzkow** [comment]
Thanks @lenasong.

On the EXIF: I can see date created for the screenshots you uploaded in the Mac OS photos app etc. but it looks to me like this is just pulling the data modified / created info from the file: Oct 1, 2018 1:47pm. Maybe all that's ever recorded is the file info? I tried taking another screenshot -- I have data on the date and time it was taken in the photos app but this again matches the file creation date -- not sure if they're decoupled or not. In any case, it would be good to resolve the lack of additional metadata is just a property of all photos, a property of screenshots, or something that is actually related to passing it through Qualtrics.

Basically, though, plan sounds good.
